manifests/mod/authz_core.pp

```diff
@@ -0,0 +1,10 @@
+# @summary
+#   Installs `mod_authz_core`.
+# 
+# @see https://httpd.apache.org/docs/current/mod/mod_authz_core.html for additional documentation.
+#
+class apache::mod::authz_core {
+  apache::mod { 'authz_core':
+    id => 'authz_core_module',
+  }
```

Comment:
`id` seems to [default to](https://github.com/puppetlabs/puppetlabs-apache/blob/abb84105bd3dcce5ba0d0ffa8e664e9e8fc5134d/manifests/mod.pp#L73) `${title}_module`, so we can strip it:

```suggestion
  apache::mod { 'authz_core':
  }
```